litmus-portal/cluster-k8s-manifest.yml

```diff
@@ -530,6 +530,14 @@ spec:
       labels:
         component: litmusportal-server
     spec:
+      initContainers:
+        - name: wait-for-mongodb
+          image: curlimages/curl:latest
```

Comment:
can you check alpine curl image?

Reply:
Sure

Reply:
@rajdas98 Currently alpine image containing curl are not multi-arch, This is only multi-arch. We will add one image in litmuschaos repo for curl with multi-arch support.